chia/protocols/wallet_protocol.py

```diff
@@ -178,29 +183,13 @@ class RespondHeaderBlocks(Streamable):
     header_blocks: List[HeaderBlock]
 
 
-@streamable
```

Comment:
I would leave this code here but commented. The protocol file acts as a sort of documentation of how wallet and node communicate.